wordpress-mobile/gutenberg-mobile
# Update gutenberg ref after react version upgrade to 16.13.1

Changes from https://github.com/WordPress/gutenberg/pull/21289 are merged to master and they create a diff in `package-lock.json` of gutenberg-mobile. This PR commits this diff and updates the gutenberg ref to current master. Also removed `react-dom` and `react-test-renderer` from `gutenberg-mobile/package.json` as they are included via `gutenberg`'s `package.json`

To test: 
1. Clean `node_modules` in gutenberg-mobile and gutenberg
1. Run `npm install` from gutenberg-mobile
1. Check that there is no diff in `package-lock.json`

PR submission checklist:
